pyIcePAP/communication.py

```diff
@@ -253,13 +253,14 @@ def send_binary(self, ushort_data):
                                           str_maskedchksum, str_data)
         self._send_data(str_bin, wait_answer=False)
 
-    def _start_thread(self):
+    def _start_thread(self, wait=True):
         self.log.debug('Start thread {0}'.format(self._connect_thread))
-        self._connect_thread = Thread(target=self._try_to_connect)
+        self._connect_thread = Thread(target=self._try_to_connect,
+                                      args=[wait])
         self._connect_thread.setDaemon(True)
         self._connect_thread.start()
 
-    def _try_to_connect(self):
+    def _try_to_connect(self, wait=True):
         self._connected = False
         sleep_time = self._timeout / 10
```

Comment:
We should divide by 10.0 (float) to ensure non integer division here.

Reply:
Good point. I will fix it